corehq/apps/hqwebapp/templates/hqwebapp/includes/bootstrap3/core_stylesheets.html

```diff
@@ -6,10 +6,10 @@
           rel="stylesheet"
           media="all"
           href="{% static 'hqwebapp/less/bootstrap.less' %}" />
-    <link type="text/less"
+    <link type="text/css"
           rel="stylesheet"
           media="all"
-          href="{% static 'font-awesome/less/font-awesome.less' %}" />
+          href="{% static '@fortawesome/fontawesome-free/css/all.min.css' %}" />
```

Comment:
Is `@fortawesome` correct, or should that be `@fontawesome`?

Reply:
this is correct. New massive js projects now have these `@` libraries that are the "main library" with all the plugin libraries beneath it. Font Awesome's is correctly FoRt Awesome lol